config/recurring.yml

```diff
@@ -7,6 +8,9 @@ default: &default
     command: SolidQueue::Job.clear_finished_in_batches
     queue: default
     schedule: at 4pm every day
+  clear_solid_queue_finished_jobs:
```

Comment:
Do we have 2 recurring task now doing the same thing?

Reply:
I've removed the other